translations/base-fr.yaml

```diff
@@ -1227,56 +1227,56 @@ tips:
+            Augmentez la surface ou supprimez-les.
+        autoComplete: Votre puzzle se complète automatiquement ! Veuillez vous assurer que vos producteurs constants
+             ne livrent pas directement à vos accepteurs d'objectifs.
 backendErrors:
-    ratelimit: You are performing your actions too frequent. Please wait a bit.
-    invalid-api-key: Failed to communicate with the backend, please try to
-        update/restart the game (Invalid Api Key).
-    unauthorized: Failed to communicate with the backend, please try to
-        update/restart the game (Unauthorized).
-    bad-token: Failed to communicate with the backend, please try to update/restart
-        the game (Bad Token).
-    bad-id: Invalid puzzle identifier.
-    not-found: The given puzzle could not be found.
-    bad-category: The given category could not be found.
-    bad-short-key: The given short key is invalid.
-    profane-title: Your puzzle title contains profane words.
-    bad-title-too-many-spaces: Your puzzle title is too short.
-    bad-shape-key-in-emitter: A constant producer has an invalid item.
-    bad-shape-key-in-goal: A goal acceptor has an invalid item.
-    no-emitters: Your puzzle does not contain any constant producers.
-    no-goals: Your puzzle does not contain any goal acceptors.
-    short-key-already-taken: This short key is already taken, please use another one.
-    can-not-report-your-own-puzzle: You can not report your own puzzle.
-    bad-payload: The request contains invalid data.
-    bad-building-placement: Your puzzle contains invalid placed buildings.
-    timeout: The request timed out.
+    ratelimit: Vous effectuez vos actions trop fréquemment. Veuillez attendre un peu s'il vous plait.
+    invalid-api-key: Échec de la communication avec le backend, veuillez essayer de
+        mettre à jour/redémarrer le jeu (clé Api invalide).
+    unauthorized: Échec de la communication avec le backend, veuillez essayer de
+        mettre à jour/redémarrer le jeu (non autorisé).
+    bad-token: Échec de la communication avec le backend, veuillez essayer de mettre à jour/redémarrer
+        le jeu (Mauvais jeton).
+    bad-id: Identifiant de puzzle non valide.
+    not-found: Le puzzle donné n'a pas pu être trouvé.
+    bad-category: La catégorie donnée n'a pas pu être trouvée.
+    bad-short-key: La clé courte donnée n'est pas valide.
+    profane-title: Le titre de votre puzzle contient des mots profanes.
+    bad-title-too-many-spaces: Le titre de votre puzzle est trop court.
+    bad-shape-key-in-emitter: Un producteur constant a un élément invalide.
+    bad-shape-key-in-goal: Un accepteur de but a un élément invalide.
+    no-emitters: Votre puzzle ne contient aucun producteur constant.
+    no-goals: Votre puzzle ne contient aucun accepteur de but.
+    short-key-already-taken: Cette touche courte est déjà prise, veuillez en utiliser une autre.
```

Comment:
"Cette touche courte" must be replace by "Cette clé" because "touche" refer to a keyboard, not a code that allows access to something